dapr/serializers/util.py

```diff
@@ -16,8 +16,9 @@
 import re
 from datetime import timedelta
 
-# Regex to parse Go Duration datatype, e.g. 4h15m50s
-DAPR_DURATION_PARSER = re.compile(r'((?P<hours>\d+)h)?((?P<mins>\d+)m)?((?P<seconds>\d+)s)?$')
+# Regex to parse Go Duration datatype, e.g. 4h15m50s123ms345μs
+DAPR_DURATION_PARSER = re.compile(
+    r'((?P<hours>\d+)h)?((?P<mins>\d+)m)?((?P<seconds>\d+)s)?((?P<milliseconds>\d+)ms)?((?P<microseconds>\d+)μs)?$')  # noqa: E501
```

Comment:
Could you add support for `us` in addition to `μs`? Technically golang durations support either (https://pkg.go.dev/time#ParseDuration)